policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/first_nations_tax/rates.yaml

```diff
@@ -0,0 +1,9 @@
+description: Abatement rate under the Yukon First nations Tax.
```

Comment:
Please phrase as something like "Yukon shifts this portion of the income tax paid by residents of settlement land to First Nations." (my vague interpretation after scanning it)